backend/src/Edna/Analysis/FourPL.hs

```diff
@@ -66,8 +66,7 @@ instance FromJSON Params4PL where
       constr (a, b, c, d) = Params4PL a b c d
 
 instance ToSchema Params4PL where
-  declareNamedSchema Proxy =
-    declareNamedSchema @Params4PLTuple Proxy
+  declareNamedSchema Proxy = declareNamedSchema @Params4PLTuple Proxy
```

Comment:
BTW, why do you derive this schema from a tuple and not from record (using generic mechanism)? Is it more convenient for front to operate on tuples?

I suppose the question is for @gromakovsky

Reply:
Hm, yes, I think I assumed that it would be easier for frontend to handle tuples, but I don't remember why I assumed that and maybe it was wrong assumption. Or maybe I decided to use tuples to have smaller JSON :shrug: 

@worm2fed if you want, you can check how this value is handled by frontend and maybe update it if you want.

Reply:
ok